### DISCUSSION TITLE
Welcome to ZoeDepth Discussions!

### DISCUSSION THREAD

**Discussion by shariqfarooq123** [isl-org/ZoeDepth#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lawrren** [comment]
python sanity_hub.py

ModuleNotFoundError: No module named 'zoedepth'

This error is haunting me because I know it's something I'm overlooking. Any help would be appreciated.

**Comment by Lumabrik** [comment]
I dont know if this is your issue, but when I unzipped the extension and put it the A1111 extensions folder the default folder name from  the zip is :  a1111-sd-zoe-depth-main
I was getting errors when trying to run the extension until I changed to folder to:  a1111-sd-zoe-depth

**Comment by Zyjer888** [comment]
Hi, I was wondering if the number on the 16-bit depth map generated from the HFface website is in millimeters and represents its distance from the lens?
